### DISCUSSION TITLE
Why isn't gaps built-in with i3?

### DISCUSSION THREAD

**Discussion by AbdelrahmanDwedar** [i3/i3#5316] [Q&A] [answered]
Most window managers have gaps either built-in or optional, and a lot of i3 users use another clone of i3 that contains gaps.

Then why don't we just afd that optional feature within the original one instead, or maybe have the ability to implement that in without the need for another clone, like a pre-built script that we can use as the window manager startup and that script for example adds that gaps.

I'd be an amazing update if you decided to do it as it'll make i3 by itself without the need for other clones better in the eyes of a lot of people that cares about how the window manager looks & feels.

**Comment by Airblader** [comment, marked as answer]
This has already happened and will be in the next release.

**Reply by AbdelrahmanDwedar** [reply]
GREAT!! What an amazing news thanks.
